src/scene_battle_rpg2k.cpp

```diff
@@ -107,7 +107,8 @@ void Scene_Battle_Rpg2k::RefreshCommandWindow() {
 
 void Scene_Battle_Rpg2k::SetState(Scene_Battle::State new_state) {
 	previous_state = state;
-	state = new_state;
+	if (new_state != State_SelectEnemyTarget && new_state != State_SelectAllyTarget)
```

Comment:
> 2.10: RM2000: When a skill or object is used, the skill/object windo… 

What I really dislike here is that this adds a filter to "SetState" which means SetState won't work as expected for all types of states. There must be a way to handle this in a better way.

Reply:
The one causes rebase hell too with the change of indentation,.